### PR TITLE
Add NaN handling to ComputeStats hook

### PR DESCRIPTION
**Pull request by FloopCZ**
- Add nanmean, nanfraction and nancount aggregators.

### CODE REVIEW

File: cxflow/tests/hooks/compute_stats_test.py [hunk 1]
```diff
@@ -27,12 +31,15 @@ def test_raise_on_init(self):
     def test_compute_save_stats(self):
         """Tests correctness of computed aggregations and their saving."""
 
-        variables = ['loss', {'accuracy': ['mean', 'std', 'min', 'max', 'median']}]
+        variables = ['loss',
+                     {'accuracy': ['mean', 'std', 'min', 'max', 'median',
+                                   'nanmean', 'nanfraction', 'nancount']},
+                     {'nan_accuracy': ['nanmean', 'nanfraction', 'nancount']}]
```
Comment (blazekadam) [inline cxflow/tests/hooks/compute_stats_test.py]: I would test if regular `'mean'` returns `NaN`...

Comment (FloopCZ) [reply cxflow/tests/hooks/compute_stats_test.py]: That is a good idea, I will do it.

**Comment by blazekadam** [comment]
Btw, there are more `nan...` functions such as `nanmax` or `nanmin`. Please, consider the dirty `getattr(np, ...)` trick.

**Comment by FloopCZ** [comment]
Ok, that could probably come in handy as well.

**Comment by FloopCZ** [comment]
@blazekadam The feedback has been addressed. Thank you for your review!

File: cxflow/hooks/compute_stats.py [hunk 2]
```diff
@@ -64,21 +66,20 @@ def _compute_aggregation(aggregation: str, data: Iterable[Any]):
...
-            return np.median(data)
-        raise ValueError('Aggregation `{}` is not supported.'.format(aggregation))
+        if aggregation == 'nanfraction':
+            return np.sum(np.isnan(data)) / len(data)
+        if aggregation == 'nancount':
+            return int(np.sum(np.isnan(data)))
+        try:
+            return getattr(np, aggregation)(data)
+        except AttributeError:
+            raise ValueError('Aggregation `{}` is not a NumPy function or a member '
```
Comment (blazekadam) [inline cxflow/hooks/compute_stats.py]: Remove duplicity with `__init__` if-raise.

**Pull request merged**